src/bt_peer_connection.cpp

```diff
@@ -3730,22 +3729,22 @@ namespace {
 			// the payload ranges they represent have been sent
 			auto first_to_keep = m_payloads.begin();
 
-			for (auto i = m_payloads.begin(); i != m_payloads.end(); ++i)
+			for (auto& m_payload : m_payloads)
```

Comment:
this is not a good name for an element in `m_payloads`

Reply:
payload better?

Reply:
I think since the scope is small, a short name is ok. like `p`

Reply:
or `i`